marquez_airflow/extractors/bigquery_extractor.py

```diff
@@ -21,6 +21,9 @@
 from marquez_airflow.extractors import BaseExtractor, StepMetadata, Source
 from marquez_airflow.extractors.sql.experimental.parser import SqlParser
 
+BIGQUERY_CONN_URL = \
```

Comment:
_Minor_: Mind adding a quick note on why we are hardcoding the connection URL for BigQuery?

Reply:
Done.